src/event_handlers/hubV1.ts

```diff
@@ -90,19 +90,56 @@ Hub.Signup.handlerWithLoader({
 // ## TRANSFERS ##
 // ###############
 
-PersonalCRC.Transfer.handler(
-  async ({ event, context }) =>
-    await handleTransfer({
+PersonalCRC.Transfer.handlerWithLoader({
+  loader: async ({ event, context }) => {
```

Comment:
interesting that we didn't have this code elsewhere before?

Reply:
which code? The `Token` and `AvatarBalance` were loaded in the handleTransfer method.

Reply:
i meant the code block below, but i guess it's because you have to load entities